fedimint-server/src/consensus/engine.rs

```diff
@@ -251,11 +252,15 @@ impl ConsensusEngine {
             self.keychain.peer_count().into(),
             self.keychain.peer_id().to_usize().into(),
             session_index,
-            max_round,
+            self.cfg
+                .consensus
+                .broadcast_rounds_per_session
+                .checked_add(EXP_SLOWDOWN_ROUNDS)
+                .expect("Rounds per session exceed maximum of u16::Max - EXP_SLOWDOWN_ROUNDS"),
             delay_config,
-            Duration::from_secs(100 * 365 * 24 * 60 * 60),
+            Duration::from_secs(10 * 365 * 24 * 60 * 60),
```

Comment:
nit: Should probably be a constant, denotes the time it would take for a session to reach max rounds, i.e. 10 years.